docs/content/en/docs/topics/repair/_index.md

```diff
@@ -22,23 +22,34 @@ The requirement for your environment may vary considerably, however the general
 1. Kubernetes cluster with the following elements deployed:
    * [K8ssandra]({{< ref "getting-started#install-k8ssandra" >}}) Helm chart
    * [Ingress Controller]({{< ref "ingress" >}})
-2. DNS name configured for the Reaper interface, referred to as _REPAIR DOMAIN_ below.
+2. DNS name configured for the Reaper web interface
+
+{{% alert title="Tip" color="success" %}}
+As an alternative to configuring an Ingress, consider port forwarding. It's another way to provide external access to resources that have been deployed by K8ssandra in your Kubernetes environment. Those resources could include Prometheus metrics, pre-configured Grafana dashboards, and the Reaper web interface for repairs of Cassandra&reg; data. The `kubectl port-forward` command does not require an Ingress/Traefik to work. 
+
+* Developers, see [Set up port forwarding]({{< ref "/docs/getting-started/developer/#set-up-port-forwarding" >}}).  
+* Site reliability engineers, see [Configure port forwarding]({{< ref "/docs/getting-started/developer/#set-up-port-forwarding" >}}).
+{{% /alert %}}
 
 ## Access the Reaper web interface
 
 ![Reaper UI](reaper-main-ui.png)
 
 With the prerequisites satisfied the Reaper web interface should be available at the following address:
 
-http://REPAIR_DOMAIN/webui
+http://REAPER_DOMAIN/webui
 
 For example, to upgrade a previously installed `k8ssandra` that's running locally, where the `releaseName` used on the prior `helm install` command was `k8ssandra`:
 
-`helm upgrade k8ssandra k8ssandra/k8ssandra --set ingress.traefik.enabled=true --set ingress.traefik.repair.host=repair.localhost`
+```bash
+helm upgrade k8ssandra k8ssandra/k8ssandra --set reaper.ingress.enabled=true , reaper.ingress.host=localhost
+```
 
-Notice how in this example, the DNS host name is specified on the command line as `repair.localhost`.
+{{% alert title="Tip" color="success" %}}
+Notice how in the `helm upgrade` example above, with Ingress/Traefik access, the DNS hostname is specified on the command line as `localhost`. If you are not running locally, specify a hostname other than `localhost`. 
+{{% /alert %}}
 
-After a few minutes, check that the pods are running. Example:
+After about five or more minutes, check that the pods are running. Example:
```

Comment:
Are we interested in showing the user how to wait programmatically while monitoring progress?

`kubectl rollout status statefulset ${CLUSTERNAME}-${DATACENTER}-default-sts`

Assuming they've followed instructions without deviation, that would be:

`kubectl rollout status statefulset k8ssandra-dc1-default-sts`

If they run it _very_ quickly after `helm upgrade` or `helm install`, they may get an error that no statefulset with that name exists; in that case they should just run it again. The command will wait up to ten minutes (the timeout can be configured longer if desired) and will give a line of output as each pod in the set becomes ready.

Reply:
Great tips, thanks Jake.